package.json

```diff
@@ -146,13 +150,6 @@
     "node": "12 - 14"
   },
   "repository": "https://github.com/github/docs",
-  "standard": {
```

Comment:
I'm curious about thoughts on including the eslint config and ignore in package.json instead of as separate files.

Reply:
I believe the separate ignore file usually has better performance, but I've not looked at using the inline config in package.json before